lmsweb/views.py

```diff
@@ -147,31 +154,79 @@ def exercises_page():
     )
 
 
+def _create_comment(session_id: int, solution: Solution):
+    user = User.get_or_none(User.id == session_id)
+    if user is None:
+        # should never happen, we checked session_id == solver_id
+        return fail(404, 'No such user')
+
+    kind = request.form.get('kind', '')
+    if (not kind) or (kind not in ('id', 'text')):
+        return fail(400, "Invalid kind")
+
+    try:
+        line_number = int(request.form.get('line', 0))
+    except ValueError:
+        line_number = 0
+    if line_number <= 0:
+        return fail(422, f"Invalid line number: {line_number}")
+
+    if kind == 'id':
+        comment_id = int(request.form.get('comment', 0))
+
+    elif kind == 'text':
+        text = request.form.get('comment', '')
+        if not text:
+            return fail(422, 'Empty comments are not allowed')
+            # TODO(LOW): Check if line number > MAX_SOLUTION_LINE_NUMBER
+        comment_id = CommentText.create(text=text).id
+    else:
+        # should never happend, kind was checked before
+        return fail(400, "Invalid kind")
+
+    Comment.create(
+        commenter=user,
+        timestamp=datetime.now(),
+        line_number=line_number,
+        comment=comment_id,
+        solution=solution
+    )
+    return jsonify({"success": "true"})
```

Comment:
Return the fields `id` and `text` with the right values. The frontend depends on it to update the view.